x-pack/plugins/inference/common/output/is_output_complete_event.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { OutputEventType, OutputCompleteEvent, OutputEvent } from '.';
+
+import type { ToolOptions } from '../chat_complete/tools';
+
+export function isOutputCompleteEvent<TId extends string, TToolOptions extends ToolOptions<string>>(
```

Comment:
NIT: same, `isOutput****Event` don't need their dedicated file for a line functions.